jupyter-notebooks/subscriptions_api_tutorial/LST_PV_Sub_Phoenix_AZ.ipynb

```diff
@@ -0,0 +1,660 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# 330 ≈ 134.33°F\n",
+    "\n",
+    "# Calculate the mean and standard deviation\n",
+    "mean_value = df[\"lst.band-1.mean\"].mean()\n",
+    "std_dev = df[\"lst.band-1.mean\"].std()\n",
+    "\n",
+    "# Define the threshold as 3 standard deviations below the mean\n",
+    "# 350K ≈ 134.33°F\n",
+    "anomaly_threshold = 335\n",
+    "\n",
+    "# Find anomalies\n",
+    "anomalies = df[df[\"lst.band-1.mean\"] > anomaly_threshold]\n",
+    "anomalies\n",
+    "\n",
+    "# Plot the line chart\n",
+    "df.title = subscription_desc[\"name\"]\n",
+    "\n",
+    "ax = df.plot.line('local_solar_time', 'lst.band-1.mean', title=df.title, figsize=(10, 6))\n",
+    "\n",
+    "# Scatter plot for anomalies\n",
+    "ax.scatter(anomalies.local_solar_time, anomalies[\"lst.band-1.mean\"], color='red', label='Anomaly')\n",
+    "\n",
+    "# Add labels to the anomalies\n",
+    "for index, row in anomalies.iterrows():\n",
+    "    ax.annotate(f'{row[\"lst.band-1.mean\"]}\\n{row[\"local_solar_time\"].strftime(\"%Y-%m-%d\")}', (row['local_solar_time'], row['lst.band-1.mean']), \n",
+    "                xytext=(5, 10), textcoords='offset points', color='red')\n",
```

Comment:
```suggestion
    "xytext=(5, 10), textcoords='offset points', arrowprops=dict(arrowstyle="simple", connectionstyle="arc3,rad=-0.2"), color='red')\n",
```

experimenting with the arrows to prevent text overlap

Reply:
not sure why the coloring is weird here...